.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 jobs:
   lint:
     docker:
-      - image: circleci/node:12
+      - image: circleci/node:14
```

Comment:
`circleci/node` is [legacy](https://circleci.com/blog/announcing-our-next-generation-convenience-images-smaller-faster-more-deterministic/), I suggest to use `cimg/node:14.17`

Reply:
Will update - good catch

Any reason for using 14.17 over the latest version?

Reply:
Because you [need to specify](https://github.com/CircleCI-Public/cimg-node#tagging-scheme) the minor version, and 14.17 is currently the latest

Reply:
Okay, I'll update the PR :)